resource-managers/kubernetes/integration-tests/src/test/resources/volcano/disable-queue.yml

```diff
@@ -19,6 +19,6 @@ kind: Queue
 metadata:
   name: queue
 spec:
-  weight: 0
+  weight: 1
   capability:
-    cpu: "1"
+    cpu: "0.1"
```

Comment:
`cpu: 0` is also invalid by definition?

Reply:
cap 0 is valid, if capacity <= 0 represents [no limit](https://github.com/volcano-sh/volcano/blob/6ea23412696eb7369e8e3dcf543604a5341058e1/pkg/scheduler/plugins/proportion/proportion.go#L104-L112) in Volcano queue.

This is different with `Weight`, weight is soft constraint to calculate some proportions according to weight value, so `0` bring some unexpected behavior, so it's invalid.

Reply:
Got it. If then, could you use more smaller value like `0.0001` than `0.1`?

Reply:
Sure, unit of least precision is 0.001 as kuberentes. So I just set to `0.001`.
 
PR updated.